sensor/kubernetes/fake/flows.go

```diff
@@ -120,6 +121,30 @@ func generateIP() string {
 	return fmt.Sprintf("10.%d.%d.%d", rand.Intn(256), rand.Intn(256), rand.Intn(256))
 }
 
+func isPublicIP(ip string) bool {
+	return net.ParseIP(ip).IsPublic()
+}
+
+func generateExternalIP() string {
+	for {
+		ip := fmt.Sprintf("%d.%d.%d.%d", rand.Intn(256), rand.Intn(256), rand.Intn(256), rand.Intn(256))
+
+		if isPublicIP(ip) {
+			return ip
+		}
+	}
```

Comment:
We might be able to simplify slightly by generating from a known-public subnet (any subnet that isn't 10.0.0.0/8, 192.168.0.0/16, or 172.16.0.0/12)

```suggestion
		return fmt.Sprintf("12.%d.%d.%d", rand.Intn(256), rand.Intn(256), rand.Intn(256))
```

Reply:
I made a similar change, but with a greater number of possible IP address.
```
// Generate IP addresses from 11.0.0.0 to 171.255.255.255 which are all public
func generateExternalIP() string {
        return fmt.Sprintf("%d.%d.%d.%d", rand.Intn(171)+11, rand.Intn(256), rand.Intn(256), rand.Intn(256))
}
```